tools/code-generation/generator/src/main/java/com/amazonaws/util/awsclientgenerator/transform/C2jModelToGeneratorModelTransformer.java

```diff
@@ -108,6 +108,8 @@ public class C2jModelToGeneratorModelTransformer {
         LEGACY_SERVICE_IDS.add("dynamodbstreams");
     }
 
+    private static Set<Integer> RESPONSE_CODES_TO_RETRY = ImmutableSet.of(500, 502, 503, 504);
```

Comment:
nit: add a comment on those magical numbers, even if it seems really obvious

Reply:
added